delft3dworker/permissions.py

```diff
@@ -1,5 +1,11 @@
 from rest_framework import permissions
 
+class HasObjectViewPermission(permissions.BasePermission):
```

Comment:
Would advise to rename this permission related to the specific action and model involved.